client/src/index.jsx

```diff
@@ -1,5 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 import ProductImagesService from './modules/app';
 
-ReactDOM.render(<ProductImagesService productId={5} />, document.getElementById('productImagesService'));
+
+ReactDOM.render(
+  <Router>
+    <div>
+      <Route path="/products/:id(\d+)" component={ProductImagesService} />
```

Comment:
This works. Another way to consider is working with window location path name rather than react router (but this works perfect just as Tandy was talking about)